frontend/src/components/Calendar/DayHeaders.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { days } from '../../common/dates'
+
+const DayContainer = styled.div`
+  font-weight: 600;
+  font-size: 13px;
+  text-align: center;
+`
+
+export const DayHeaders = () => {
+  return (
+    <>
+      {days.slice(1).map((day) => {
+        return <DayContainer key={1}>{day.slice(0, 3)}</DayContainer>
+      })}
+    </>
```

Comment:
<> no longer needed! 

return days.slice.....

Reply:
![Screenshot 2022-02-10 at 11 15 05 AM](https://user-images.githubusercontent.com/60415251/153330354-38c9dda1-c29e-4ace-97a2-3bed7139af15.png)
This gave error. Is this what you were referring to?